styles/styles.css

```diff
@@ -934,7 +906,13 @@ main .schedule-demo .default-content-wrapper h2 {
 
 /* Position preceding content */
 .section.arc-after-section {
-  margin-top: -20%;
+  margin-top: -13%;
+  position: relative;
+}
+
+/* stylelint-disable-next-line no-descending-specificity */
+.section.arc-after-section p { 
+  text-align: left;
```

Comment:
Please do not use `stylelint-disable` and move it to the corresponding location in the css